definitions/tools/mark_duplicates_and_sort.cwl

```diff
@@ -21,9 +21,9 @@ requirements:
             declare MD_BARCODE_TAG
             if [ ! -z "${5}" ]; then
               MD_BARCODE_TAG="BARCODE_TAG=${5}"
```

Comment:
If you're adding the sortorder as a param, the above two need to be changed to `$6`